<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="TechAltum-PC" name="Test Case 2" tests="1" failures="1" timestamp="28 Feb 2015 08:43:12 GMT" time="21.588" errors="0">
  <testcase name="atTest" time="2.425" classname="myPack1.TC02">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at myPack1.TC02.atTest(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- atTest -->
</testsuite> <!-- Test Case 2 -->
